<html>
<head>
 <meta charset="utf-8">
 <meta http-equiv="Content-Type" content="text/html">
 <meta name="keywords" content="Modeling, Simulation, COPASI, Conference, ODE-based dynamic modelling, parameter estimation, quantitative proteomics, lactic acid bacteria, primary energy metabolism">

 
  <!-- Favicon and touch icons -->
    <link rel="shortcut icon" href="/images/ico/favicon.ico">
    <link rel="apple-touch-icon-precomposed" sizes="144x144" href="/images/ico/apple-touch-icon-144-precomposed.png">
    <link rel="apple-touch-icon-precomposed" sizes="114x114" href="/images/ico/apple-touch-icon-114-precomposed.png">
    <link rel="apple-touch-icon-precomposed" sizes="72x72" href="/images/ico/apple-touch-icon-72-precomposed.png">
    <link rel="apple-touch-icon-precomposed" href="/images/ico/apple-touch-icon-57-precomposed.png">
    
   <link rel="stylesheet" type="text/css" href="/stylesheets/bootstrap-3.3.4/css/COPASI-bootstrap.css">
    
    <title>CMC16 - Paper 24</title>
</head>
<body>
<div class="container">
  <div class="row">
    <div class="col-xs-12">

    <p align="center"><a href="/Events/CMC16/"><b>2016 Conference on Computational Modelling with COPASI</b></a> <br/>
    Manchester Institute of Biotechnology, 12<sup>th</sup> &ndash; 13<sup>th</sup> May, 2016</p>

    <h2 align="center">Combining dynamic modelling and quantitative proteomics</h2>
    
    <h3 align="center"><u>Nadine Veith</u><sup>1</sup>, Johan van Heerden<sup>2</sup>, Ching Chiek Koh<sup>3</sup>, Tomas Fiedler<sup>4</sup>, Bernd Kreikemeyer<sup>4</sup>, Jeroen Hugenholtz<sup>5</sup>, Ruedi Aebersold<sup>3</sup>, Ursula Kummer<sup>1</sup></h3>

   <p>1 - Heidelberg University, Germany; 2 - Vrije Universiteit Amsterdam, The Netherlands; 3 - ETH ZÃ¼rich, Switzerland; 4 - University of Rostock, Germany; 5 - University of Amsterdam, The Netherlands</p> 

   <p><b>Keywords:</b> ODE-based dynamic modelling, parameter estimation, quantitative proteomics, lactic acid bacteria, primary energy metabolism</p>
   
   <h3>Abstract</h3>
   <p>The lactic acid bacterium, Enterococcus faecalis, relies almost exclusively on glycolysis and fermentation for energy production. This suggests that targeting its primary energy metabolism may lead to perturbation of cell growth and proliferation. In this study, we investigated the primary energy metabolism of this opportunistic human pathogen. First, we constructed an ODE-based dynamic model to describe the glucose uptake, glycolysis, fermentation and export of fermentation "end-"products using COPASI. As kinetic parameters are only sparsely available for this organism, we derived unknown kinetic constants by fitting the model to experimentally determined metabolite concentrations using parameter estimation. The metabolite data comprised selected extra- and intracellular metabolites measured in a time-course experiment after a glucose pulse.</p>
   <p>To better understand the primary energy metabolism of Enterococcus faecalis, we combined dynamic modelling with quantitative proteomics. We used a mass spectrometric technique, PCT-SWATH, to measure proteomic changes during the glucose pulse. Using spiked-in synthetic peptides of known concentrations, we quantified selected "key-"enzymes of the primary energy metabolism in absolute concentrations during the time-course experiment and observed intriguingly high enzyme concentrations. In order to integrate the dynamic changes of enzyme abundances, we further constructed a small model describing phenomenologically the gene expression and protein biosynthesis of the quantified "key-"enzymes. The model parameters were derived by parameter estimation while fitting the model to dynamic changes of the measured enzyme abundances.</p>
   <p>Finally, we combined the two models. First analyses of the extended model suggests that processes that are directly involved in glucose uptake are most sensitive.</p>
   <p><a href="/Events/CMC16/Program">Conference Program</a></p>
  </div>
</div>
</div>
</body>
</html>